frontend/src/pages/Login.tsx

```diff
@@ -1,3 +1,4 @@
+import.meta.hot;
```

Comment:
where is this used? or why is this import needed?

Reply:
when reading the env variable using hot reload this is needed, otherwise react fails to read the environment